catalog/internal/utils.go

```diff
@@ -170,7 +170,7 @@ func getDefaultWarehouseLocation(dbname, tablename string, nsprops, catprops ice
 	}
 
 	if warehousepath := catprops.Get("warehouse", ""); warehousepath != "" {
-		return url.JoinPath(warehousepath, dbname+".db", tablename)
+		return url.JoinPath(warehousepath, dbname, tablename)
```

Comment:
as far as I'm aware, iceberg-python unconditionally adds the `.db` suffix, should we make this an option instead?

Reply:
i looked at rust one, lemme check how it works with py

Reply:
yeah, make sense to make this conditional, i'll think about how to pass it there.